config/environments/.env.dev

```diff
@@ -1,6 +1,5 @@
 APP_ENV="dev"
```

Comment:
Should standardize environment names with the rest. I.e;
```
APP_ENV="development"
```

Reply:
Above is changed